client/src/components/search/searchBar/SearchSuggestion.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Highlight } from 'react-instantsearch-dom';
-import { isEmpty } from 'lodash';
 
 const Suggestion = ({ hit, handleMouseEnter, handleMouseLeave }) => {
-  const dropdownFooter = hit.objectID.includes('default-hit-');
-  return isEmpty(hit) || isEmpty(hit.objectID) ? null : (
+  const dropdownFooter = hit.objectID.includes('footer-');
+  const noHits = hit.title === 'No tutorials found';
```

Comment:
I don't think it's a good idea to have the suggestion's behaviour dependant on the precise text of the title.  It should be possible to change the title without breaking behaviour - i.e. it shouldn't break if we use a localised version.

How about creating a separate component, `NoHitsSuggestion` for instance, and just using that in `SearchHits`?

Reply:
Very good point, thank you @ojeytonwilliams.

Sure, please feel free to refactor this and create a separate component if you have some time. I'll be quite busy with some new curriculum and News related things for the next day or two.

Reply:
I won't be able to until next week, sorry. The PR looks good otherwise, though, so no need to delay.

Reply:
Alright, no problem. Thank you as always for your review and great suggestions. I agree that we should refactor this to use a "no hits" component so this solution isn't so brittle, especially with the upcoming localizations.